dask/dataframe/io/parquet/core.py

```diff
@@ -911,6 +912,18 @@ def get_engine(engine):
         return eng
 
     elif engine in ("pyarrow", "arrow", "pyarrow-legacy", "pyarrow-dataset"):
+
+        pa = import_required("pyarrow", "`pyarrow` not installed")
+        pa_version = parse_version(pa.__version__)
+
+        if engine in ("pyarrow", "arrow"):
```

Comment:
Do we have a test for this logic engine selection logic?

Reply:
We should also update the description of `engine` in `read_parquet` / `to_parquet` to match these updates